DATA/production/qc-async/hmp.json

```diff
@@ -0,0 +1,53 @@
+{
+   "qc":{
+      "config":{
+         "database":{
+            "implementation":"CCDB",
+            "host":"ccdb-test.cern.ch:8080",
+            "username":"not_applicable",
+            "password":"not_applicable",
+            "name":"not_applicable"
+         },
+         "Activity":{
+            "number":"42",
+            "type":"2"
+         },
+         "monitoring":{
+            "url":"infologger:///debug?qc"
+         },
+         "consul":{
+            "url":"http://consul-test.cern.ch:8500"
+         },
+         "conditionDB":{
+            "url":"ccdb-test.cern.ch:8080"
+         }
+      },
+    "tasks": {
+         "TaskClusters":{
+            "active":"true",
+            "className":"o2::quality_control_modules::hmpid::HmpidTaskClusters",
+            "moduleName":"QcHMPID",
+            "detectorName":"HMP",
+            "cycleDurationSeconds":"60",
+            "maxNumberCycles":"-1",
+            "dataSource":{
+               "type":"direct",
+               "query":"intrecord:HMP/INTRECORDS1/0;clusters:HMP/CLUSTERS/0"
+            }
+         },
+         "TaskMatches":{
```

Comment:
The same comment as above the task. Please refer to https://github.com/AliceO2Group/QualityControl/blob/master/doc/ModulesDevelopment.md#naming-convention